cluster/ci/config/prow/plugins.yaml

```diff
@@ -266,6 +266,9 @@ plugins:
   - wip
   - yuks
 
+  heketi/heketi:
```

Comment:
Have we enabled webhook delivery yet?

Reply:
Not yet. I should have sufficient rights to do it any time we're ready though. Please let me know if you think that we're close enough to a usable config that we should set it up.